test/system/pages_test.rb

```diff
@@ -326,4 +326,20 @@ class PagesTest < ApplicationSystemTestCase
     end
     assert_equal '.file-input', find('textarea.a-text-input')['data-input']
   end
+
+  test 'check if Docs are sorted by update order' do
+    practice = practices(:practice1)
+    page1 = Page.create!(title: '三番目に新しい更新のDocs', body: 'test', user: users(:komagata), practice:, updated_at: Time.current,
+                         published_at: Time.current)
+    page2 = Page.create!(title: '二番目に新しい更新のDocs', body: 'test', user: users(:komagata), practice:, updated_at: Time.current,
+                         published_at: Time.current)
+    page3 = Page.create!(title: '一番新しい更新のDocs', body: 'test', user: users(:komagata), practice:, updated_at: Time.current, published_at: Time.current)
```

Comment:
テストの追加ありがとうございます！
よりよくするためにいくつかコメントさせていただきます🙏

- 今回証明したいことは「更新順に並んでいること(`updated_at`)」なので、注目しない`published_at`の記述は要らないと思いました。テストデータがそれぞれ何をやっているかが一目でわかりやすくなります👍
- 3つとも`updated_at`が`Time.current`ですが、Flakyなテストの原因になってしまいます。[全て同時刻の場合、DBに保存されるタイミングによってランダムな並び順になるためです。](https://github.com/fjordllc/bootcamp/pull/8196#pullrequestreview-2440360744)

これらを踏まえて、以下のように修正するのがいいかと思います〜🙆

```suggestion
user = users(:komagata)
now = Time.current

page1 = Page.create!(title: '三番目に新しい更新のDocs', body: '三番目に新しいです。', user:, practice:, updated_at: now + 1.hour)
page2 = Page.create!(title: '二番目に新しい更新のDocs', body: '二番目に新しいです。', user:, practice:, updated_at: now + 2.hours)
page3 = Page.create!(title: '一番新しい更新のDocs', body: '一番新しいです。', user:, practice:, updated_at: now + 3.hours)
```

Reply:
`updated_at`の部分はどうしようか悩んでいたところでした。その他の細かい部分も勉強になります！
ありがとうございます<( _ _ )>